composer.json

```diff
@@ -44,6 +44,7 @@
         "symfony/web-link": "5.3.*",
         "symfony/webpack-encore-bundle": "^1.11",
         "symfony/yaml": "5.3.*",
+        "symfonycasts/verify-email-bundle": "^1.7",
         "twig/extra-bundle": "^2.12|^3.0",
         "twig/twig": "^2.12|^3.0"
     },
```

Comment:
Hello, I don't understand why we have this file here, It should not be in this PR

Reply:
same for the composer.lock